test/e2e/mock-response-data/snaps/snap-binaries-and-headers/bip32-example-snap-2.3.0-headers.json

```diff
@@ -0,0 +1,7 @@
+{
+  "Accept-Ranges": "bytes",
+  "Content-Length": "72040",
+  "Content-Type": "application/octet-stream",
+  "Etag": "\"f6aefd54643480c1b753fa4244a537cf\"",
+  "Vary": "Accept-Encoding"
+}
```

Comment:
I extracted the response headers in separate files, in order to be able to update them whenever we download a new snap version. So for every snap update, we'll get 1 file for the binary and another file for the relevant needed headers